src/www/diag_logs_settings.php

```diff
@@ -399,7 +399,8 @@ function check_everything() {
                       <div class="hidden" for="help_for_logfilesize">
                         <?=gettext("Logs are held in constant-size circular log files. This field controls how large each log file is, and thus how many entries may exist inside the log. By default this is approximately 500KB per log file, and there are nearly 20 such log files.") ?>
                         <br /><br />
-                        <?=gettext("NOTE: Log sizes are changed the next time a log file is cleared or deleted. To immediately increase the size of the log files, you must first save the options to set the size, then clear all logs using the \"Reset Log Files\" option farther down this page. "); ?>
+                        <strong><span class="text-danger"><?=gettext("NOTE:");?></span></strong>
```

Comment:
I found it under System - Networking
Several "Notes" are orange (see screenshot)
But the first "Note" at "Allow IPv6" is black.
Wouldn't it be better if all notes would be in orange?

![networking_settings_system](https://cloud.githubusercontent.com/assets/11537653/12074176/6b018804-b146-11e5-83c4-d54f833e180e.png)